closed/OpenJ9.gmk

```diff
@@ -379,14 +384,15 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
   CMAKE_ARGS += $(EXTRA_CMAKE_ARGS)
 
   ifeq (true,$(OPENJ9_ENABLE_CUDA))
-    CMAKE_ARGS += -DJ9VM_OPT_CUDA=ON -DOMR_CUDA_HOME="$(CUDA_HOME)"
+    CMAKE_ARGS += -DJ9VM_OPT_CUDA=ON -DOMR_CUDA_HOME="$(call UnixPath,$(CUDA_HOME))"
   else # OPENJ9_ENABLE_CUDA
     CMAKE_ARGS += -DJ9VM_OPT_CUDA=OFF
   endif # OPENJ9_ENABLE_CUDA
 
 $(OUTPUTDIR)/vm/cmake.stamp :
 	@$(MKDIR) -p $(@D)
-	cd $(@D) && $(CMAKE) $(CMAKE_ARGS) $(OPENJ9_TOPDIR)
+	cd $(@D) && export $(EXPORT_MSVS_ENV_VARS) > /dev/null \
+		&& $(CMAKE) $(CMAKE_ARGS) $(OPENJ9_TOPDIR)
```

Comment:
This should not use `export`, eliminating the need to ignore errors; it should be:
```
cd $(@D) && $(EXPORT_MSVS_ENV_VARS) $(CMAKE) $(CMAKE_ARGS) $(OPENJ9_TOPDIR)
```

Reply:
That is what I initially tried but I was getting the same errors about unidentified compilers. Although now I am suspecting its an unrelated intermittent issue